tests/testthat/test-rapporttekst-funksjoner.R

```diff
@@ -113,3 +143,44 @@ test_that("akse_tall_format() gir ut en tom tekstvektor hvis den tar
   tall_format = akse_tall_format()
   expect_identical(tall_format(integer()), character())
 })
+
+# Testing av prosent()
+
+test_that("prosent() gir ut tankestrek når «x»-argumentet inneholder NA", {
+  expect_identical(prosent(NA_real_), "\\textendash{}")
+})
+
+test_that("prosent() gir ut riktig verdi når kun «x»-argumentet er gitt", {
+  expect_identical(prosent(numeric()), character(0))
```

Comment:
Denne linja kunne godt vera skilt ut som éigen test, dvs. ein test på at funksjonen handterer inndata av lengd 0. Då får ein ei meir nyttig feilmelding.

Viss ein har ei naiv implementering av funksjonen som noko slikt (som fungerer nesten alltid):

```r
paste0(num(x), "\\prosent")
````

er det forvirrande å få ei feilmelding som seier at testen «prosent() gir ut riktig verdi når kun «x»-argumentet er gitt» feila. For funksjonen *vil* jo oftast fungera når kun `x`-argumentet er oppgjeve. Det er mykje betre å få ei feilmelding som seier at testen «prosent() gir ut riktig verdi ved tom inndata» (elns.) feila.